private-templates-service/adaptivecards-templating-service/src/util/templateutils.ts

```diff
@@ -153,3 +151,12 @@ export function isValidJSONString(input: string) {
   }
   return true;
 }
+
+/**
+ * Get token from authorization header.
+ * @param token 
+ */
+export function parseToken(token: string): string {
+  let bearer = token.split(/[ ]+/).pop();
```

Comment:
Should this be unit tested in case structure of token changes. Just a thought...

Reply:
Yep, I'll add some more unit tests for multiple users in a follow up PR and will include tests for malformed tokens/authorization headers!